plugins/content-no-strings/content-no-strings.test.js

```diff
@@ -0,0 +1,39 @@
+const testRule = require('stylelint-test-rule-tape');
+const shopifyi18n = require('.');
+
+testRule(shopifyi18n.rule, {
+  ruleName: shopifyi18n.ruleName,
+  config: true,
+  skipBasicChecks: true,
```

Comment:
Oh, it's back. = )

Reply:
yeah haha I just realized what it did: it adds a lot of noise to the test output that doesn't seem relevant
<img width="419" alt="image" src="https://user-images.githubusercontent.com/7321311/42222094-266c1552-7ea2-11e8-9895-c39c0e46dde5.png">